src/AddLink.jsx

```diff
@@ -0,0 +1,19 @@
+import linkImg from './assets/link.svg';
+
+import './AddLink.css';
+
+const AddLink = () => {
+  return (
+    <div className='container-add-link'>
+      <div className='add-link'>
+        <div className='add-link_icon-box'>
+          <img src={linkImg} alt='링크'></img>
+          <span className='text-add-link'>링크를 추가해 보세요</span>
+        </div>
+        <button className='add-link_button'>추가하기</button>
```

Comment:
버튼의 type attribute는 디폴트가 submit이라서 form에서 사용되는게 아니라면 명시적으로 작성해주는게 좋을 것 같아요.
```suggestion
        <button type='button' className='add-link_button'>추가하기</button>
```

Reply:
수정 후, form 태그 안에 button 요소가 들어가 있습니다. 그럴 경우, 타입을 명시하지 않아도 되나요?

Reply:
네~ 그렇게 되면 디폴트값이 submit이기 때문에 지금은 명시하지 않아도 좋습니다!